Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM docker.io/python:3.10
+
+# Create dirs for:
+# - Injecting config.yml: /root/.DANE
+# - Mount point for input & output files: /mnt/dane-fs
+# - Storing the source code: /src
+RUN mkdir /root/.DANE /mnt/dane-fs /src /data
+
+WORKDIR /src
+
+ENV POETRY_NO_INTERACTION=1 \
+    POETRY_VIRTUALENVS_IN_PROJECT=1 \
+    POETRY_VIRTUALENVS_CREATE=1 \
+    POETRY_CACHE_DIR=/tmp/poetry_cache
+
+COPY pyproject.toml poetry.lock ./
```

Comment:
Just a minor improvement would be to move this copy statement after installing poetry (also in the example worker, in fact)

Reply:
Could you add it maybe to the PR you opened for the example worker?